db/schema.rb

```diff
@@ -459,8 +458,8 @@
     t.integer "price_group_id", null: false
     t.integer "schedule_rule_id", null: false
     t.decimal "discount_percent", precision: 10, null: false
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
```

Comment:
This seems odd

Reply:
Seems correct: https://github.com/rails/rails/issues/44571#issuecomment-1054758649